cmd2/cmd2.py

```diff
@@ -3592,12 +3569,14 @@ def _generate_transcript(self, history: List[Union[HistoryItem, str]], transcrip
             self.perror('Failed to save transcript: {}'.format(ex), traceback_war=False)
         else:
             # and let the user know what we did
-            if len(history) > 1:
+            if commands_run > 1:
                 plural = 'commands and their outputs'
             else:
                 plural = 'command and its output'
             msg = '{} {} saved to transcript file {!r}'
-            self.pfeedback(msg.format(len(history), plural, transcript_file))
+            self.pfeedback(msg.format(commands_run, plural, transcript_file))
+
+        return stop
```

Comment:
I'm not sure if I'd want generating a transcript to quit the overall cmd2 application.  But I'm also not sure I wouldn't.  I guess its a judgement call.

Reply:
I feel pretty strongly that generating a transcript shouldn't stop the application. I think it's the wrong behavior, and it's a change from current behavior.

Reply:
Just as a quick input: when run in interactive mode, I don't want it to stop the application. Yet I have a bunch of scripts that only generates transcript and if they fail, I would like to know about it (having cmd2 return an error).

Reply:
I do not think generating a transcript should ever stop the application, a stop should just stop the transcript generation loop.  If it stops early, it would probably also be a good idea to print a warning message to the user letting them know which command caused it to stop early.

Reply:
Should this warning message be printed with `perror` or `pfeedback`?

Reply:
Transcript generation now just stops and prints a message if all commands have not run